pkgs/development/python-modules/sentry-sdk/default.nix

```diff
@@ -55,8 +81,11 @@ buildPythonPackage rec {
   postCheck = "unset NIX_REDIRECTS LD_PRELOAD";
 
   # no tests
-  doCheck = false;
-  pythonImportsCheck = [ "sentry_sdk" ];
+  doCheck = true;
```

Comment:
`doCheck = !stdenv.isDarwin;`
Tests are broken on darwin.

Reply:
Thanks for the information.